.gitignore

```diff
@@ -6,3 +6,4 @@ coverage
 .idea
 .env
 config/meta.dev.json
+pkg/api/src/meta.dev.json
```

Comment:
@mturley, It makes sense to leave `config/meta.dev.json` unchanged to avoid any impact on the controller.
Which doesn't preclude to move some of the configuration belonging to pkg/web in its realm.

Reply:
We can remove this from gitignore now if the file isn't there anymore.